core/chains/evm/txmgr/common.go

```diff
@@ -35,25 +35,33 @@ func batchSendTransactions(
 	reqs := make([]rpc.BatchElem, len(attempts))
 	ethTxIDs := make([]int64, len(attempts))
 	hashes := make([]string, len(attempts))
+	now := time.Now()
+	successfulBroadcast := []int64{}
 	for i, attempt := range attempts {
 		ethTxIDs[i] = attempt.TxID
 		hashes[i] = attempt.Hash.String()
+		signedTx, decodeErr := GetGethSignedTx(attempt.SignedRawTx)
+		if decodeErr != nil {
+			return reqs, now, successfulBroadcast, fmt.Errorf("failed to decode signed raw tx into Transaction object: %w", decodeErr)
+		}
+		txBytes, marshalErr := signedTx.MarshalBinary()
```

Comment:
Could you explain via a comment, why do we have to decode the raw tx and encode it again before sending in batched req?